content/06.methods.md

```diff
@@ -125,7 +126,7 @@ This workflow is available in the [D3b GitHub repository](https://github.com/d3b
 ##### SNV and indel calling
 
 For PBTA samples, we used four variant callers to call SNVs and indels from panel, WXS, and WGS data: `Strelka2` [@doi:10.1038/s41592-018-0051-x], `Mutect2` [@biorxiv:10.1101/861054], `Lancet` [@doi:10.1038/s42003-018-0023-9], and `VarDictJava` [@doi:10.1093/nar/gkw227].
-`VarDictJava`-only calls were not retained since ~ 39M calls with low VAF were uniquely called and may be potential false positives.
+`VarDictJava`-only calls were not retained since ~ 39M calls with low VAF were uniquely called and may be potential false positives (~1.2M calls were included in consensus SNV calling described below).
```

Comment:
```suggestion
`VarDictJava`-only calls were not retained since ~ 39M calls with low VAF were uniquely called and may be potential false positives (~1.2M calls from Mutect2, Strelka2, and Lancet were included in consensus SNV calling described below).
```

thanks for clarifying - can we update to this to show this is excluding those 39M?

Reply:
I don't think "and/or" is appropriate here because a call only gets included if it is present 3 out of 3 of the stated callers?

```suggestion
`VarDictJava`-only calls were not retained since ~ 39M calls with low VAF were uniquely called and may be potential false positives. (~1.2M calls were called by `Mutect2`, `Strelka2`, and `Lancet` and included consensus CNV calling as described below.)
```

Reply:
Oh yes you are right, I was thinking about hotspot as well.